src/main/java/com/develhope/spring/users/components/UserMapper.java

```diff
@@ -1,18 +1,29 @@
 package com.develhope.spring.users.components;
 
 import com.develhope.spring.users.dtos.UserCreatorDTO;
-import com.develhope.spring.users.dtos.UserSavedDTO;
+import com.develhope.spring.users.dtos.UserResponseDTO;
+import com.develhope.spring.users.dtos.UserReworkedDTO;
+import com.develhope.spring.users.dtos.UserUpdaterDTO;
 import com.develhope.spring.users.models.User;
 import org.springframework.stereotype.Component;
 
 @Component
 public class UserMapper {
 
-    public UserSavedDTO toUserSavedDTOFrom(User user) {
-        return new UserSavedDTO(user.getName(), user.getSurname(), user.getRoles());
+    public UserResponseDTO toResponseDTO(User user) {
+        return new UserResponseDTO(user.getId(), user.getName(), user.getSurname(), user.getRoles());
     }
 
-    public User toEntityFrom(UserCreatorDTO userDTO) {
+    public UserCreatorDTO toCreatorDTO(User user) {
+        return new UserCreatorDTO(user.getName(), user.getSurname(), user.getPhoneNumber(),user.getEmail(),user.getRoles());
+    }
+
+    public UserUpdaterDTO toUserUpdaterDTO (User user){
+        return new UserUpdaterDTO(user.getId(), user.getName(), user.getSurname(), user.getPhoneNumber(),
+                user.getEmail(), user.getRoles());
+    }
+
+    public User toEntity(UserCreatorDTO userDTO) {
```

Comment:
Mi sfugge perchè qua ritorniamo una enitity con solo due dati

Reply:
Non ha 2 dati, devi espandere la visione. ci sono il nome, il cognome, il numero (che dovremo cambiare in string), l'email e il ruolo

Reply:
Non ha 2 dati, devi espandere la visione. ci sono il nome, il cognome, il numero (che dovremo cambiare in string), l'email e il ruolo